ibis/backends/duckdb/__init__.py

```diff
@@ -1589,6 +1589,24 @@ def _get_schema_using_query(self, query: str) -> sch.Schema:
             }
         )
 
+    def _should_cache_physical_table(self, op: ops.PhysicalTable) -> bool:
+        if isinstance(op, (ops.DatabaseTable, ops.UnboundTable)):
```

Comment:
This feels like a property we should encode on the operation itself instead of leaving it up to the backend.

Is that not possible for some reason?